### ISSUE TITLE
Action reports success even if the domain could not be checked

### ISSUE THREAD

**Issue by iamsilvio** [bluenovaio/action-tls-monitor#24]
### Description

Action reports success even if the domain could not be checked! 
Action output see repro ->

### Reproduction

```
Run bluenovaio/action-tls-monitor@main
  with:
    domain: deleteonerror.com
    expiration_days: 30
    approved_protocols: TLSv1.3
(node:2076) UnhandledPromiseRejectionWarning: Error: write EPROTO 139733454043008:error:1409442E:SSL routines:ssl3_read_bytes:tlsv1 alert protocol version:../deps/openssl/openssl/ssl/record/rec_layer_s3.c:1544:SSL alert number 70

    at WriteWrap.onWriteComplete [as oncomplete] (internal/stream_base_commons.js:87:16)
(node:2076) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:2076) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```

### Environment

```
 Current runner version: '2.278.0'
Operating System
  Ubuntu
  20.04.2
  LTS
Virtual Environment
  Environment: ubuntu-20.04
  Version: 20210504.1
```

![Screenshot 2021-05-06 170328](https://user-images.githubusercontent.com/1840449/117320940-00161100-ae8d-11eb-818b-e326667a68aa.jpg)

**Comment by zrosenbauer** [comment]
@iamsilvio just to verify you are using the Slack Webhook? This Action DOES NOT error but returns the output with an error_message so you can send to PagerDuty, OpsGenie or other services if you'd like.

**Comment by zrosenbauer** [comment]
Merged #25 and released patch [1.1.2](https://github.com/bluenovaio/action-tls-monitor/releases/tag/1.1.2).

Please confirm the issue is resolved.

**Comment by iamsilvio** [comment]
> 
> 
> @iamsilvio just to verify you are using the Slack Webhook? This Action DOES NOT error but returns the output with an error_message so you can send to PagerDuty, OpsGenie or other services if you'd like.

I expected a different behavior! But if I know that , I can also work with it :)

**Comment by iamsilvio** [comment]
> 
> 
> Merged #25 and released patch [1.1.2](https://github.com/bluenovaio/action-tls-monitor/releases/tag/1.1.2).
> 
> Please confirm the issue is resolved - @iamsilvio

Your fix solved it! Thx for your work!

**Comment by zrosenbauer** [comment]
@iamsilvio if you want it to error on failure we can possibly add that as a feature. Our goal is to provide the info so you can pass it to a 3rd party system.
